ZuluContent/Items/Skill Items/Magical/Runebook.cs

```diff
@@ -209,16 +209,15 @@ public void DropRune(Mobile from, RunebookEntry e, int index)
 
             m_Entries.RemoveAt(index);
 
-            RecallRune rune = new RecallRune();
-
-            rune.Target = e.Location;
-            rune.TargetMap = e.Map;
-            rune.Description = e.Description;
-            rune.House = e.House;
-            rune.Marked = true;
-
+            var rune = new RecallRune
+            {
+                Target = e.Location, 
+                TargetMap = e.Map, 
+                Description = e.Description, 
```

Comment:
House missing?

Reply:
It's there, I just made it a computed property instead. It's only used for the rune description, not for any travel checks.

https://github.com/zuluhotelaustralia/zuluhotel/blob/7010e19bc2da96593b7bb72f14307f9ab1b022b1/ZuluContent/Items/Skill%20Items/Magical/Misc/RecallRune.cs#L13-L15